dynawo/sources/Models/Modelica/Dynawo/Electrical/Shunts/ShuntBNonLinear.mo

```diff
@@ -0,0 +1,68 @@
+within Dynawo.Electrical.Shunts;
+
+/*
+* Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model ShuntBNonLinear "Shunt element with voltage dependent reactive power and a variable susceptance given by a table and a section"
+
+  import Dynawo.Connectors;
+  import Dynawo.Types;
+  import Dynawo.Electrical.Controls.Basics.SwitchOff;
+  import Modelica;
+
+  extends SwitchOff.SwitchOffShunt;
+
+public
+
+  Connectors.ACPower terminal(V(re(start = u0Pu.re), im(start = u0Pu.im)), i(re(start = i0Pu.re), im(start = i0Pu.im))) "Connector used to connect the shunt to the grid";
+  Connectors.ZPin section(value (start = section0)) "section position of the shunt";
```

Comment:
space between "value" and parenthesis or not?

Reply:
I think it is better without the space (it is indeed clearer in OpenModelica as it highlights the text when there is not space)

Reply:
ok, thanks!